prow/pjutil/pjutil.go

```diff
@@ -37,14 +37,41 @@ import (
 	"k8s.io/test-infra/prow/pod-utils/downwardapi"
 )
 
+// Options allows a client to set some fields
```

Comment:
some?

Reply:
As of today `Options` exists to set the state only but we can't exclude someone will come and add another `Option` for something else.